specs/latest/2.0/index.html

```diff
@@ -415,6 +415,7 @@ <h2 class="no-toc">Table of contents</h2>
 typedef (Int32Array or sequence&lt;GLint&gt;) ClearBufferIVSource;
 typedef (Uint32Array or sequence&lt;GLuint&gt;) ClearBufferUIVSource;
 typedef (Float32Array or sequence&lt;GLfloat&gt;) ClearBufferFVSource;
+typedef (ArrayBuffer or SharedArrayBuffer) ByteBuffer;
```

Comment:
In the WebGL 1.0 spec, there is already a BufferDataSource typedef which takes ArrayBuffer or ArrayBufferView. I recommend modifying that to also allow SharedArrayBuffer instead of introducing a new typedef called ByteBuffer.  Besides, ByteBuffer is a bit confusing since ArrayBuffers can contain things other than bytes: floats, integers, etc, are all allowed.